### PR TITLE
Polish README.adoc

### PR DESCRIPTION
**Pull request by Abdullah8006**
Updated minor grammar mistakes in the **README.adoc** file.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -233,5 +233,5 @@ Boot step-by-step:
 
 
 == License
-Spring Boot is Open Source software released under the
+Spring Boot is an Open Source software released under the
```
Comment (snicoll) [inline README.adoc]: I believe the current phrasing is correct.

File: README.adoc [hunk 2]
```diff
@@ -102,7 +102,7 @@ need JDK 1.8.
 	$ ./gradlew publishToMavenLocal
 ----
 
-This will build all of the jars and documentation and publish every to your local
+This will build all of the jars and documentation and publish it to your local
```
Comment (snicoll) [inline README.adoc]: isn't `them` more appropriate here?

Comment (Abdullah8006) [reply README.adoc]: @snicoll true

**Comment by snicoll** [comment]
Thanks for the follow-up @Abdullah8006 and for making your first contribution to Spring Boot.

**Pull request closed without merging**